src/components/PageContent/Card/Card.jsx

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as Rivet from "../../util/Rivet";
+import { TestUtils } from "../../util/TestUtils";
+
+const Card = ({
+  children,
+  className,
+  clickable = false,
+  Component = 'div',
+  eyebrow,
+  horizontal = false,
+  image,
+  meta,
+  raised = false,
+  testMode = false,
+  title,
+  titleUrl,
+  ...attrs
+}) => {
+  const classNameArr = [
```

Comment:
Should we do any validation that if `clickable` is `true` that `title` and `titleUrl` must not be undefined?

Reply:
The title is already required from the prop types and enforcing that with code hasn't been a practice (most checks are to prevent javascript errors).  This would be similar with the titleUrl, if the url isn't set then the clickable doesn't class doesnt do anything (card acts like a normal without link).